### DISCUSSION TITLE
bg_color01 with btnmatrix

### DISCUSSION THREAD

**Discussion by Greginkansas** [HASwitchPlate/openHASP#417] [Q&A] [answered]
Can I use the bg_color01 with btnmatrix?

{"page":1,"id":5,"obj":"btnmatrix","x":2,"y":2,"w":476,"h":245,"text_font":48,"bg_color01":"#ff0004","options":["#000000 Screen Door#","#000000 Front Door#","\n","#000000 G Door#","#000000 Back Door#"],"toggle":1,"one_check":0}

**Comment by fvanroie** [comment, marked as answer]
> Can I use the bg_color01 with btnmatrix?

`bg_color01` doesn't apply because it is the background color of the *whole* brnmatrix, which doesn't have a toggled state itself.
The _individual_ buttons can have a toggled state: so use the partnumber `40` for items/buttons + `01` for toggled state:

```json
{"page":11,"id":5,"obj":"btnmatrix","x":2,"y":2,"w":476,"h":245,"text_font":48,"text_color40":"black","text_color41":"white","bg_color41":"#ff0004","options":["Screen Door","Front Door","\n","G Door","Back Door"],"toggle":1,"one_check":0}
```

**Reply by Greginkansas** [reply]
Add them got it.
thanks
